src/components/v5/shared/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,47 +1,64 @@
+import { CaretRight } from '@phosphor-icons/react';
 import clsx from 'clsx';
 import React, { type FC } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { useBreadcrumbsContext } from '~context/BreadcrumbsContext/BreadcrumbsContext.ts';
 
 import Link from '../Link/index.ts';
 
-import BreadcrumbsCardSelect from './BreadcrumbsCardSelect.tsx';
-import { type BreadcrumbsItem, type BreadcrumbsProps } from './types.ts';
+import { type BreadcrumbsProps } from './types.ts';
+import { getBreadcrumbItemName } from './utils.ts';
 
 const displayName = 'v5.Breadcrumbs';
 
-const getBreadcrumbItem = (item: BreadcrumbsItem) => {
-  if ('href' in item && item.href) {
-    return (
-      <Link to={item.href} className="text-inherit">
-        {item.label}
-      </Link>
-    );
-  }
+const Breadcrumbs: FC<BreadcrumbsProps> = ({ className }) => {
+  const { rootBreadcrumbItem, shouldShowBreadcrumbs } = useBreadcrumbsContext();
+  const location = useLocation();
 
-  if ('dropdownOptions' in item) {
-    return <BreadcrumbsCardSelect item={item} />;
+  if (!shouldShowBreadcrumbs) {
+    return null;
   }
+  const pathSections = location.pathname.split('/');
+  const breadcrumbItems = pathSections.slice(2);
 
-  return <h5>{item.label}</h5>;
-};
+  const getBreadcrumbLink = (index: number) => {
+    const rootLinkPrefix = rootBreadcrumbItem
+      ? `${rootBreadcrumbItem.link}/`
+      : '';
+
+    return `${rootLinkPrefix}${breadcrumbItems.slice(0, index + 1).join('/')}`;
+  };
```

Comment:
I think if we want to cover the paths from the user account, we should refactor like this

```suggestion
  const pathSections = location.pathname.split('/');
  const initialPath = pathSections.slice(0, 2).join('/');
  const breadcrumbItems = pathSections.slice(2);

  const getBreadcrumbLink = (index: number) => {
    return `${initialPath}/${breadcrumbItems.slice(0, index + 1).join('/')}`;
  };
```